packages/angular/projects/ui-angular/src/lib/primitives-poc/view/view-component/view.component.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  HostBinding,
+  Input,
+  OnChanges,
+  OnInit,
+  Renderer2,
+  SimpleChanges,
+  ViewEncapsulation,
+} from '@angular/core';
+
+/** Config file to get all the configuration related the strings and values */
+import { ComponentSettings } from './ComponentSetting';
+@Component({
+  selector: ComponentSettings.COMPONENTNAME,
+  template: '<ng-content></ng-content>',
+  encapsulation: ViewEncapsulation.None,
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class AmplifyViewComponent implements OnInit, OnChanges {
+  /** Input Prop for style changes  */
+  @Input() type: 'submit' | 'button' = 'button';
+  @Input() fullWidth: boolean | string = false;
+  @Input() isDisabled: boolean | string = false;
+  @Input() size: 'small' | 'medium' | 'large' = 'medium';
+  @Input() variation: 'primary' | 'default' | 'link' = 'default';
```

Comment:
This looks like props of Button and not View. (for example `type` is a button-only prop). Do you want to update the input list to resemble props on https://ui.docs.amplify.aws/react/components/view?tab=props?

(obviously, this is a POC and doesn't need all the props there, but I'd still make sure View props are here).

Reply:
Updated all the prop